### PR TITLE
Add SVG to Razor syntax /3

### LINKED ISSUE #22848: Add .net 6 SVG
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b87067b2-347c-072d-26c8-5a8c7c9c860b
* Version Independent ID: a0b3b70a-3b39-1b5f-a4ce-c41bd2d6b044
* Content: [Razor syntax reference for ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/mvc/views/razor?view=aspnetcore-5.0)
* Content Source: [aspnetcore/mvc/views/razor.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/mvc/views/razor.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-mvc**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #22848 per https://devblogs.microsoft.com/aspnet/asp-net-core-updates-in-net-6-preview-5/
[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/mvc/views/razor?view=aspnetcore-6.0&branch=pr-en-us-22853#scalable-vector-graphics-svg)

### CODE REVIEW

**Review by guardrex — APPROVED** [review]
Just line length nits.

File: aspnetcore/mvc/views/razor.md [hunk 1]
```diff
@@ -41,6 +41,28 @@ HTML attributes and content containing email addresses don't treat the `@` symbo
...
+
+::: moniker range=">= aspnetcore-6.0"
+
+### Scalable Vector Graphics (SVG)
+
+[SVG](https://developer.mozilla.org/docs/Web/SVG) [foreignObject](https://developer.mozilla.org/docs/Web/SVG/Element/foreignObject) elements are supported:
+
+```html
+@{
+    string message = "This is an example of using foreignObject with Scalable Vector Graphics (SVG)";
```
[the remark this answers is not in the document]
Comment (Rick-Anderson) [reply aspnetcore/mvc/views/razor.md:53]: But I thought the whole point of SVG was nice formatting of long blocks?

[the remark this answers is not in the document]
Comment (Rick-Anderson) [reply aspnetcore/mvc/views/razor.md:53]: I suppose that's long enough

[the remark this answers is not in the document]
Comment (guardrex) [reply aspnetcore/mvc/views/razor.md]: My understanding is that it wasn't supported at all until now. Apparently, browsers wouldn't display the text until they fixed it.

https://github.com/dotnet/aspnetcore/pull/32417

Side-note: We keep a bit of SVG-in-Blazor coverage in the Components overview. I put the Blazor SVG coverage in on https://github.com/dotnet/AspNetCore.Docs/pull/22717.

**Pull request merged**